commit a4b3675f9b838c0dc6ef680010233ba5dbebb26d
Author: Hans Verkuil <hverkuil@xs4all.nl>
Date:   Wed Nov 28 04:11:52 2018 -0500

    media: vivid: add req_validate error injection
    
    Add a new vivid button control to inject an error into the
    req_validate request callback.
    
    This will help testing with v4l2-compliance.
    
    Signed-off-by: Hans Verkuil <hverkuil-cisco@xs4all.nl>
    Signed-off-by: Mauro Carvalho Chehab <mchehab+samsung@kernel.org>

diff --git a/drivers/media/platform/vivid/vivid-core.h b/drivers/media/platform/vivid/vivid-core.h
index 0bec2c3401d2..a4ff8547f79d 100644
--- a/drivers/media/platform/vivid/vivid-core.h
+++ b/drivers/media/platform/vivid/vivid-core.h
@@ -294,6 +294,7 @@ struct vivid_dev {
 	bool				buf_prepare_error;
 	bool				start_streaming_error;
 	bool				dqbuf_error;
+	bool				req_validate_error;
 	bool				seq_wrap;
 	bool				time_wrap;
 	u64				time_wrap_offset;